app/models/federated_protocols/oidc.rb

```diff
@@ -38,6 +38,10 @@ def service_provider
       request.service_provider
     end
 
+    def enhanced_ipp_required?
```

Comment:
We have been trying to get away from including these methods on the protocol classes or adding the to the session. This was the way we would forward along SP request requirements before we built the `AuthnContextResolver` and resolved them based on VTRs and ARC values downstream.